### DISCUSSION TITLE
Welcome to sysmon Discussions!

### DISCUSSION THREAD

**Discussion by gridhead** [t0xic0der/sysmon#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MasKeo13** [comment]
I came across this project and wanted to give it a try.

Question, are there steps/insturctions for the 'Connect now' portion after launch the web Interface? How do I enable the SuperVisor driver service?

TIA.
